src/components/DesktopContainer.jsx

```diff
@@ -16,6 +16,36 @@ import { HomepageHeading } from "./HomepageHeading";
  */
 class DesktopContainer extends Component {
   state = {};
+  
```

Comment:
These members look like they're reused in the MobileContainer.jsx file - see if you can solve removing the code duplication by extracting the members and making them reusable.